### PR TITLE
[185049] log process with too many messages

### PR DESCRIPTION
**Pull request by aDaichiOta**
https://acsmine.tok.access-company.com/redmine/issues/185049

### CODE REVIEW

File: core/sup_tree_core/system_metrics_reporter.ex [hunk 1]
```diff
@@ -76,4 +79,17 @@ defmodule AntikytheraCore.SystemMetricsReporter do
       end)
     Enum.map(absolute_values ++ cumulative_values, fn {label, value} -> {label, :gauge, value} end)
   end
+
+  defp log_too_many_messages_processes() do
+    ps = :recon.proc_count(:message_queue_len, 5)
+    [{_pid, top_len, _info}|_] = ps
+    if top_len >= @log_message_size do
+      L.error("There are processes with more than #{@log_message_size} messages.")
```
Comment (irisTa56) [inline core/sup_tree_core/system_metrics_reporter.ex]:
```suggestion
      L.error("There are process(es) with more than or equal to #{@log_message_size} messages.")
```

I think the suggested message more accurately describes the code.

File: core/sup_tree_core/system_metrics_reporter.ex [hunk 2]
```diff
@@ -76,4 +79,17 @@ defmodule AntikytheraCore.SystemMetricsReporter do
...
+      Enum.each(ps, fn({pid, len, _info}) ->
```
Comment (irisTa56) [inline core/sup_tree_core/system_metrics_reporter.ex]:
```suggestion
      Enum.each(ps, fn {pid, len, _info} ->
```

To be consistent with the existing code.

File: core/sup_tree_core/system_metrics_reporter.ex [hunk 3]
```diff
@@ -76,4 +79,17 @@ defmodule AntikytheraCore.SystemMetricsReporter do
       end)
     Enum.map(absolute_values ++ cumulative_values, fn {label, value} -> {label, :gauge, value} end)
   end
+
+  defp log_too_many_messages_processes() do
+    ps = :recon.proc_count(:message_queue_len, 5)
+    [{_pid, top_len, _info}|_] = ps
```
Comment (irisTa56) [inline core/sup_tree_core/system_metrics_reporter.ex]:
```suggestion
    [{_pid, top_len, _info} | _] = ps
```

To be consistent with the existing code.

**Comment by irisTa56** [comment]
LGTM except for three suggestions!

File: core/sup_tree_core/system_metrics_reporter.ex [hunk 4]
```diff
@@ -20,7 +20,9 @@ defmodule AntikytheraCore.SystemMetricsReporter do
   alias Antikythera.ExecutorPool.Id, as: EPoolId
   alias AntikytheraCore.MetricsUploader
   alias AntikytheraCore.Vm
+  require AntikytheraCore.Logger, as: L
 
+  @log_message_size 100_000
```
Comment (aMasakiTakahashi) [inline core/sup_tree_core/system_metrics_reporter.ex]: I think `@log_message_queue_lenght` is better.
The reason is that you used `len` and `top_len` as the number of messages.

**Comment by aMasakiTakahashi** [comment]
LGTM except for one comment.

**Comment by ttokit** [comment]
LGTM!

**Pull request merged**